### PR TITLE
Add document about addon option to cppcheck

### PR DESCRIPTION
**Pull request by dev-yakuza**
Related on [sider/runners #1119](https://github.com/sider/runners/pull/1119)

### CODE REVIEW

File: docs/tools/cplusplus/cppcheck.md [hunk 1]
```diff
@@ -125,3 +126,19 @@ See also the `--project` option via the command-line help.
...
+linter:
+  cppcheck:
+    addon:
+      - cert
+      - misra
+```
+
+The available values are `cert`, `misra`, `y2038`, and `threadsafety`.
+
+See also the [`--addon`](https://github.com/danmar/cppcheck/tree/master/addons).
```
Comment (yykamei) [inline docs/tools/cplusplus/cppcheck.md:144]: The title of the specified `README.md` looks "Cppcheck addons", so how about this?

```suggestion
See also [Cppcheck addons](https://github.com/danmar/cppcheck/tree/master/addons).
```

Comment (dev-yakuza) [reply docs/tools/cplusplus/cppcheck.md:144]: It's better. I will modify it.

Thanks!

File: docs/tools/cplusplus/cppcheck.md [hunk 2]
```diff
@@ -125,3 +126,19 @@ See also the `--project` option via the command-line help.
...
+linter:
+  cppcheck:
+    addon:
+      - cert
+      - misra
+```
+
+The available values are `cert`, `misra`, `y2038`, and `threadsafety`.
+
+See also [Cppcheck addons](https://github.com/danmar/cppcheck/tree/master/addons).
```
Comment (ybiquitous) [inline docs/tools/cplusplus/cppcheck.md:144]: [suggestion] How about the following simplification?

```suggestion
This option allows you to set the [predefined addons](https://github.com/danmar/cppcheck/tree/master/addons). For example:

```yaml
linter:
  cppcheck:
    addon:
      - cert
      - misra
```

The available values are `cert`, `misra`, `y2038`, and `threadsafety`.
```

**Comment by ybiquitous** [comment]
Released! 🎉

**Pull request merged**